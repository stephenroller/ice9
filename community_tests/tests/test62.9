# simple 1D array in proc

proc foo(i : int)
	var pi : int[5]

	pi[i] := 3;
	write pi[i];
	write i;
end

foo(2);
foo(1);

