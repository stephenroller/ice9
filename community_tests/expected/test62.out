line 12: syntax error near end of file
